src/dotnet/Fable.Tools/ProjectCracker.fs

```diff
@@ -242,8 +252,9 @@ let getPaketProjRefs paketDir projFile =
             []
 
 let getProjectOptionsFromFsproj projFile =
-    let paketDir = tryFindPaketDirFromProject projFile
-    paketDir |> Option.iter checkFableCoreVersion
+    //let paketDir = tryFindPaketDirFromProject projFile
+    //paketDir |> Option.iter checkFableCoreVersion
+    let paketDir = None
```

Comment:
Why are you getting errors? Is there a project with `paket.references` but no `paket.dependencies` in a parent project? It that's the case I think it makes sense to throw an error.